src/classes/jsonld/class-jsonld-article-wrapper.php

```diff
@@ -128,12 +128,33 @@ private function is_article( $schema_types ) {
 		return ! empty( $array_intersect );
 	}
 
+	/**
+	 * Get primary author from author array.
+	 * 
+	 * A helper function that checks the structure of the author array and returns the primary author.
+	 * 
+	 * @param array|string 	$author The author array.
+	 * 
+	 * @return array|string The primary author.
+	 * 
+	 * @since 3.51.4
+	 */
+	private function get_primary_author( $author ) {
+		
+		// Nested array of co-authors. Return the primary author.
+		if ( ! isset( $author['@id'] ) ) {
```

Comment:
I think we need to add a couple more checks here, like `if ( is_array( $author) && !empty( $author ) && ! isset( $author['@id'] ) ) {` ... otherwise you risk trying to access index `0` of something which is not an array or has no index `0`.